Client/iTeamTalk/iTeamTalk/PreferencesViewController.swift

```diff
@@ -122,9 +122,6 @@ class PreferencesViewController : UITableViewController, UITextFieldDelegate, Te
         let settings = UserDefaults.standard
         
         var nickname = settings.string(forKey: PREF_GENERAL_NICKNAME)
-        if nickname == nil {
```

Comment:
A "nil" in nickname will cause a crash.

Reply:
So this functionality cannot be implemented for IOS client?